shared/components/map.js

```diff
@@ -187,7 +187,7 @@ export default class Map extends Component {
         ]
       );
       this.setState({userLocationEnabled: false});
-    });
+    }, {enableHighAccuracy: true, timeout: 20000, maximumAge: 1000});

```

Comment:
have you tested on this? I have not included this highAccuracy yet since on some android device which does not support this feature simply makes the app crashed.